core/services/bulletprooftxmanager/eth_confirmer_test.go

```diff
@@ -1510,20 +1499,20 @@ func TestEthConfirmer_RebroadcastWhereNecessary(t *testing.T) {
 		// Do the thing
 		require.NoError(t, ec.RebroadcastWhereNecessary(context.TODO(), currentHead))
 
-		etx, err = cltest.FindEthTxWithAttempts(db, etx.ID)
+		etx, err = borm.FindEthTxWithAttempts(etx.ID)
 		require.NoError(t, err)
 
 		assert.Equal(t, bulletprooftxmanager.EthTxUnconfirmed, etx.State)
 
 		// Got the new attempt
-		attempt1_4 = etx.EthTxAttempts[3]
+		attempt1_4 = etx.EthTxAttempts[0]
 		assert.Equal(t, expectedBumpedGasPrice.Int64(), attempt1_4.GasPrice.ToInt().Int64())
 
 		require.Len(t, etx.EthTxAttempts, 4)
-		require.Equal(t, attempt1_1.ID, etx.EthTxAttempts[0].ID)
-		require.Equal(t, attempt1_2.ID, etx.EthTxAttempts[1].ID)
-		require.Equal(t, attempt1_3.ID, etx.EthTxAttempts[2].ID)
-		require.Equal(t, attempt1_4.ID, etx.EthTxAttempts[3].ID)
+		require.Equal(t, attempt1_1.ID, etx.EthTxAttempts[3].ID)
+		require.Equal(t, attempt1_2.ID, etx.EthTxAttempts[2].ID)
+		require.Equal(t, attempt1_3.ID, etx.EthTxAttempts[1].ID)
+		require.Equal(t, attempt1_4.ID, etx.EthTxAttempts[0].ID)
```

Comment:
These are reversed because we now always use a consistent ordering for fetching EthTxAttempts as an association on the EthTx